src/components/FallenHeroes.tsx

```diff
@@ -11,32 +11,46 @@ function FallenHeroes() {
   // matching hero from the set of totalHeroes,
   // and is included in rest of heroes alive, update the state accordingly
   const { campaign, setCampaign } = React.useContext(LotrContext);
+
+  function filterFallen() {
+    return campaign.allHeroes.filter((hero) => hero.alive === false);
+  }
+
+  const fallen = filterFallen();
   const [fallenHero, setFallenHero] = useState("");
 
+  const names = campaign.allHeroes.map((hero) => {
+    return hero.name;
+  });
```

Comment:
This could be a one-liner. Take a look at `arrow-body-style` Eslint rule.

Reply:
true

Reply:
Add the rule to Eslint config pls and make sure Eslint works in your editor.

Reply:
added

Reply:
As it is not a syntax error, let's just emit a warning. Also, `as-needed` is the default (see [docs](https://eslint.org/docs/latest/rules/arrow-body-style)), so you can just do `"arrow-body-style": ["warn"]`.

Reply:
adjusted